std/json.d

```diff
@@ -83,20 +83,31 @@ enum JSONOptions
 /**
 JSON type enumeration
 */
-enum JSON_TYPE : byte
+enum JSONType : byte
 {
     /// Indicates the type of a `JSONValue`.
-    NULL,
-    STRING,  /// ditto
-    INTEGER, /// ditto
-    UINTEGER,/// ditto
-    FLOAT,   /// ditto
-    OBJECT,  /// ditto
-    ARRAY,   /// ditto
-    TRUE,    /// ditto
-    FALSE    /// ditto
+    null_,
+    string,   /// ditto
+    integer,  /// ditto
+    uinteger, /// ditto
+    float_,   /// ditto
+    array,    /// ditto
+    object,   /// ditto
+    true_,    /// ditto
+    false_,   /// ditto
+    NULL = null_,
+    STRING = string,
+    INTEGER = integer,
+    UINTEGER = uinteger,
+    FLOAT = float_,
+    ARRAY = array,
+    OBJECT = object,
+    TRUE = true_,
+    FALSE = false_,
```

Comment:
I would probably add comments that this starts the deprecation cycle in 2.082 (same for JSON_TYPE  below)

Reply:
added